cross/dav1d/Makefile

```diff
@@ -18,21 +18,4 @@ CONFIGURE_ARGS += -Denable_examples=false
 CONFIGURE_ARGS += -Denable_docs=false
 ADDITIONAL_CFLAGS = -O
 
-include ../../mk/spksrc.archs.mk
-
-# Define x86asm
-ifeq ($(findstring $(ARCH),$(i686_ARCHS) $(x64_ARCHS)),$(ARCH))
-BUILD_DEPENDS = native/nasm
-NASM_PATH = $(abspath $(PWD)/../../native/nasm/work-native/install/usr/local/bin)
-ENV += AS=$(NASM_PATH)/nasm
-ENV += PATH=$(NASM_PATH):$$PATH
-CONFIGURE_ARGS += -Denable_asm=true
-
-# Allow ASM on aarch64, disable on all others
-else ifeq ($(findstring $(ARCH),$(ARMv8_ARCHS)),$(ARCH))
-CONFIGURE_ARGS += -Denable_asm=true
-else
-CONFIGURE_ARGS += -Denable_asm=false
```

Comment:
nasm is only used for x86/x86_64, while for aarch64 it uses the gcc assembler (`as`).  Considering the code above, you should have kept something such as:
```
ifeq ($(findstring $(ARCH),$(ARMv8_ARCHS) $(i686_ARCHS) $(x64_ARCHS)),$(ARCH))
CONFIGURE_ARGS += -Denable_asm=true
else
CONFIGURE_ARGS += -Denable_asm=false
endif
```
While it probably builds ok, I recall code to fail on armv7 in multiple cases which led to enable assembly optimizations only on aarch64 and x86 archs.  I would suggest keeping this ifeq else usecase (and maybe elsewhere, haven't looked at all the code changes).

EDIT: On another comment I realized that issues with `as` on armv7 related to gcc <= 4.9, rather suggesting:

```
include ../../mk/spksrc.common.mk

ifeq ($(findstring $(ARCH),$(ARMv8_ARCHS) $(i686_ARCHS) $(x64_ARCHS)),$(ARCH))
CONFIGURE_ARGS += -Denable_asm=true

# only enable `asm` on armv7 when using gcc > 4.9.* (i.e. DSM7+)
else if ($(findstring $(ARCH),$(ARMv7_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
ifeq ($(call version_lt, ${TC_GCC}, 5),1)
CONFIGURE_ARGS += -Denable_asm=true
endif
else
CONFIGURE_ARGS += -Denable_asm=false
endif
```